### ISSUE TITLE
uca1400_* collations

### ISSUE THREAD

**Issue by shionryuu** [adminerevo/adminerevo#50]
alter database, table do not show uca1400_* collations

version `Adminer 4.8.2 MySQL 11.0.3-MariaDB-1:11.0.3+maria~ubu2204`

**Comment by Lumeriol** [comment]
Yes, I can confirm, that problem exists with used MariaDB v10.10 and higher.

Problem is, that uca1400 collations are for null charset, that is in Adminer transformed to empty string as key for array of collations. And for select box of Collations is created array with empty string used for "prompt" value with translation and merged with collations array, so uca1400 is dropped.

**Comment by sebbu2** [comment]
The issue is worse than that. I configured MariaDB to use uca1400 collation by default on the server, so including database connection and client.
adminerevo straight up refuse to connect:
`Server sent charset (0) unknown to the client. Please, report to the developers`
